### DISCUSSION TITLE
Pimoroni Pico Wireless ESP32-WROOM32E support

### DISCUSSION THREAD

**Discussion by Invictaz** [darthcloud/BlueRetro#660] [Ideas]
Pimoroni made an add-on board (a hat) for the Raspberry Pi Pico to add storage, wireless (if you don't have a Pico W) in a Espressif ESP-WROOM-32E chipset. I hope Blueretro can be ported because the Pico is a cheap solution for Blueretro.

https://www.kiwi-electronics.com/nl/pico-wireless-pack-10680?country=NL&utm_term=10680&gclid=EAIaIQobChMI_ZL9n733_QIVr4ODBx10agKmEAQYGCABEgKHmPD_BwE

**Comment by mknrls** [comment]
The esp32 could be easily flash with bluereto but the problem you would have is the pinout, some are are use for connect the pico to the esp32. 

Second problem is Pimoroni is being discontinued the product (EOL).  

Although the idea of having the pico and esp32 work together could be interesting!

**Comment by Invictaz** [comment]
Discontinued should not be a problem. The NES is discontinued by Nintendo and still new innovations are released for it. 

The pinout would not be that difficult since only The data likes to the pico are used right?
